### PR TITLE
Add sauce credentials

### PR DESCRIPTION
**Pull request by plehegar**
For https://github.com/w3c/web-platform-tests/pull/5866

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5920)

### CODE REVIEW

**Comment by jgraham** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/web-platform-tests/5920)*

**Review by jgraham — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +22,8 @@ install:
   - pip install -U setuptools
   - pip install -U requests
 env:  # required at the top-level for allow_failures to work below
+global:
```
Comment (bobholt) [inline .travis.yml]: This should be inset:
```
env:
  global:
    SAUCE_USERNAME=w3c_ttwf
```

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232415514)

# Firefox (nightly) #
Testing web-platform-tests at revision 4103c1d79579c506e80610f10093173c3d51f30e
Using browser at version BuildID 20170515100238; SourceStamp e66dedabe582ba7b394aee4f89ed70fe389b3c46
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232415520)

# Chrome (unstable) #
Testing web-platform-tests at revision 4103c1d79579c506e80610f10093173c3d51f30e
Using browser at version 60.0.3095.5 dev
Starting 10 test iterations

**Review by bobholt — CHANGES_REQUESTED** [review]
Just an indentation issue.

This doesn't add the jobs, which is fine - I can update my PR, and land that after this.

**Review by bobholt — APPROVED** [review]

**Pull request merged**